.github/workflows/progress-tracker.yml

```
@@ -0,0 +1,126 @@
+name: Progress Tracker
+
+env:
+  # Update these environment variables every year
+  FILTER_LABEL: '2020 chapter'
+  TRACKER_ISSUE_NUMBER: 989
```

Not loving the picking up of number of issues from first issue. Means the whole thing breaks if first issue is broken (like it is on my fork at the mo!). Also it took me a while to figure out what was going on as missed your comment, so wonder if we'll remember how this works next year.

I think we should add as a param instead:

```suggestion
  FILTER_LABEL: '2020 chapter'
  TRACKER_ISSUE_NUMBER: 989
  TOTAL_TRACKED_TASKS: 10
```

---

Parameterizing it would make some of the code logic simpler, but there will be more things to update. That's why I tried to make it auto recognize. It would have been a lot more flexible if we had parsable short badges for each task, then we could have used a hash instead of an array and filled up any empty sports with not applicable and they could be reordered as well. Something like:

* [x] *LEAD* - Jul 5 - A lead selected
* [ ] *TEAM* - Jul 15 - At least one member in each team
* [ ] *OUTLINE* - Aug 3 - Chapter outline ready

Then we could use these short names as task identifiers in the column rather than relying on sequence number.

#JustSharingMyThoughts :wink:

---

True, but that makes the issues less readable.

I think what you’ve done is pretty awesome and will be very useful. I’m only suggesting this extra change to make it easier to understand next year as took me a bit to understand. Plus I think it makes it more robust to that first issue being edited so it no longer follows the pattern. So I think it’s worth the extra config.